README.md

```diff
@@ -35,5 +35,12 @@ A Persian (Farsi) Font
            url('Samim-Bold.ttf') format('truetype');
       font-weight: bold;
     }
+    
+    body {
+        /* optional enhancements for better rendering */
+        text-rendering: auto;
+        -webkit-font-smoothing: antialiased;
+        -moz-osx-font-smoothing: grayscale;
```

Comment:
It works for icon fonts most of the time since they don't have tiny stokes. The `antialiased` causes the text to appear slimmer and blurry on HiDPI screens (unlike the old CRT and low density LCD monitors) and it's really bad for fonts with tiny and cursive stokes like Arabic alphabet.